src/js/apps/globals/nav_app.js

```diff
@@ -214,7 +214,7 @@ export default App.extend({
     this.showChildView('navContent', navView);
   },
   showSearch(prefillText) {
-    if (!Radio.request('bootstrap', 'currentOrg:setting', 'patient_search_settings')) {
+    if (!Radio.request('bootstrap', 'currentOrg:setting', 'patient_search')) {
```

Comment:
Changed this and all other instances of `patient_search_settings` to `patient_search`.